config.js

```diff
@@ -438,6 +432,35 @@ config.DEFAULT_ACCOUNT_PREFERENCES = {
 ///////////////////////////////////
 config.REMOTE_NOOAA_NAMESPACE = `remote-${config.KUBE_APP_LABEL}`;
 
+///////////////////////////////
+// FILES RELATED             //
+///////////////////////////////
+config.INLINE_MAX_SIZE = 4096;
+
+// Bucket cache expiration time
+config.BUCKET_CACHE_EXPIRY_MS = 60000;
```

Comment:
Didn't we remove this new cache?

Reply:
It is used at https://github.com/noobaa/noobaa-core/blob/aaea2a15dc6e0682b3907a20547834b915dda77b/src/sdk/object_sdk.js#L30. Since it is taken care of by https://github.com/noobaa/noobaa-core/blob/aaea2a15dc6e0682b3907a20547834b915dda77b/src/sdk/object_sdk.js#L156, I will rename it to NAMESPACE_CACHE_EXPIRY and change wherever using NAMESPACE_CACHE_EXPIRY to get it from config.

Reply:
I see but this cache was actually intentionally using `expiry_ms: 0` and used an item validation function instead that already handled the time expiry.
https://github.com/noobaa/noobaa-core/blob/67c2e94f5d60c1b9774f669b5435b1dd98d6079a/src/sdk/object_sdk.js#L144-L158

So what we should have is a comment over that `expiry_ms: 0` saying that we do not use expiry_ms because our validation function is handling expiry on its own.

Reply:
Sure. I will add comment.